code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -252,6 +256,25 @@ Please contact me on #coderbus IRC. ~Carn x
 		list_huds = hud_list.Copy()
 		list_huds += backplane // Required to mask HUDs in context menus: http://www.byond.com/forum/?post=2336679
 
+//TYPING INDICATOR CODE.
+	if(client && !stat) //They have a client & aren't dead/KO'd? Continue on!
+		if(typing_indicator && hud_typing) //They already have the indicator and are still typing
+			overlays += typing_indicator //This might not be needed? It works, so I'm leaving it.

```

Comment:
Not needed and defeats the entire purpose of using a mutable appearance. Will get immediately erased when the ma is applied anyway. Don't do this.